README.md

```diff
@@ -320,7 +320,7 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
   ilab download
   ```
 
-  `ilab download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace and store it in a `models` directory:
+  By default, `ilab download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace and store it in a `models` directory:
```

Comment:
When we say "the model" we are talking about Merlinite yes? Should we explicitly call that out here?